content/docs/tasks/policy-enforcement/rate-limiting/index.md

```diff
@@ -187,58 +171,59 @@ so the configuration to enable rate limiting on both adapters is the same.
       - name: request-count
         namespace: istio-system
       services:
-      - name: ratings
-        namespace: default
-      - name: reviews
-        namespace: default
-      - name: details
-        namespace: default
       - name: productpage
         namespace: default
+      # - service: '*'
     {{< /text >}}
 
     This `QuotaSpecBinding` binds the `QuotaSpec` you created above to the
-    services you want to apply it to. You have to define the namespace for
-    each service since it is not in the same namespace this `QuotaSpecBinding`
-    resource was deployed into.
+    services you want to apply it to. `productpage` is explicitly bound to `request-count`, note
+    that you must define the namespace since it differs from the namespace of the `QuotaSpecBinding`.
+    If the last line is uncommented, `service: '*'` binds all services to the `QuotaSpec`
+    making the first entry redundant.
 
 1. Refresh the product page in your browser.
 
-    * If you are logged out, `reviews-v3` service is rate limited to 1 request
-    every 5 seconds. If you keep refreshing the page, the stars should only
-    load around once every 5 seconds.
-
-    * If you log in as user "jason", `reviews-v2` service is rate limited to 5
-    requests every 10 seconds. If you keep refreshing the page, the stars
-    should only load 5 times every 10 seconds.
-
-    * For all other services, the default 5000 qps rate limit will apply.
+    `request-count` quota applies to `productpage` and it permits 2 requests
+    every 5 seconds. If you keep refreshing the page you should see
+    `RESOURCE_EXHAUSTED:Quota is exhausted for: requestcount`.
 
 ## Conditional rate limits
 
-In the previous example you applied a rate limit to the `ratings` service
-without regard to non-dimension attributes. It is possible to conditionally
-apply rate limits based on arbitrary attributes using a match condition in
-the quota rule.
+In the above example we have effectively rate limited `productpage` at `2 rps` per client IP.
+Consider a scenario where you would like to exempt clients from this rate limit if a user is logged in.
```

Comment:
This paragraph could use a little rework to reorder to remove "Consider a scenario where" but not going to block on it.